StreamChatUI.podspec

```
@@ -1,25 +1,23 @@
 Pod::Spec.new do |spec|
-    spec.name = "StreamChatUI"
-    spec.version = "4.5.0"
-    spec.summary = "StreamChat UI Components"
-    spec.description = "StreamChatUI SDK offers flexible UI components able to display data provided by StreamChat SDK."
+  spec.name = "StreamChatUI"
+  spec.version = "4.5.0"
+  spec.summary = "StreamChat UI Components"
+  spec.description = "StreamChatUI SDK offers flexible UI components able to display data provided by StreamChat SDK."
 
-    spec.homepage = "https://getstream.io/chat/"
-    spec.license = { :type => "BSD-3", :file => "LICENSE" }
-    spec.author = { "getstream.io" => "support@getstream.io" }
-    spec.social_media_url = "https://getstream.io"
-    spec.swift_version = "5.2"
-    spec.platform = :ios, "11.0"
-    spec.source = { :git => "https://github.com/GetStream/stream-chat-swift.git", :tag => "#{spec.version}" }
-    spec.requires_arc = true
+  spec.homepage = "https://getstream.io/chat/"
+  spec.license = { :type => "BSD-3", :file => "LICENSE" }
+  spec.author = { "getstream.io" => "support@getstream.io" }
+  spec.social_media_url = "https://getstream.io"
 
-    spec.source_files = "Sources/StreamChatUI/**/*.swift"
-    spec.exclude_files = ["Sources/StreamChatUI/**/*_Tests.swift", "Sources/StreamChatUI/**/*_Mock.swift"]
-    spec.resource_bundles = { "StreamChatUIResources" => ["Sources/StreamChatUI/Resources/**/*"] }
+  spec.swift_version = "5.2"
```

Our CI uses Swift 5.5 atm. Dunno if we're supporting also Swift version 5.2

```suggestion
  spec.swift_version = "5.5"
```

---

I was trying to do as little changes as possible. Maybe this is something we should first debate internally?

---

ok I see your point ;)